packages/node/test/unit/plugins/operator/heartbeatUtils.test.ts

```diff
@@ -14,6 +15,21 @@ describe('heartbeatUtils', () => {
         expect(() => HeartbeatMessageSchema.parse(msg)).not.toThrow()
     })
 
+    it('messages without version pass validation', () => {
+        const msg = omit(
+            createHeartbeatMessage({
+                nodeId: 'nodeId',
+                websocket: {
+                    port: 31313,
+                    host: '127.0.0.1',
+                    tls: false
+                }
+            }),
+            'version'
+        )
```

Comment:
No `omit` needed if you do

```ts
const { version: _, ...msg } = createHeartbeatMessage(…)
```

Cheers!

Reply:
Using `omit` is simpler and more straightforward as we don't need version stored into a variable.